torchvision/transforms/functional_tensor.py

```diff
@@ -0,0 +1,32 @@
+import torch
+import torchvision.transforms.functional as F
+
+def vflip(img_tensor):
+    """Vertically flip the given the Image Tensor.
+
+    Args:
+        img_tensor (Tensor): Image Tensor to be flipped.
```

Comment:
Can you specify in the documentation what is the expected format of the tensor? Is it `HxWxC` or `CxHxW`?